src/lib/storage/column_encoders/deprecated_dictionary_encoder.hpp

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include <algorithm>
+#include <limits>
+#include <memory>
+
+#include "base_column_encoder.hpp"
+
+#include "storage/deprecated_dictionary_column.hpp"
+#include "storage/fitted_attribute_vector.hpp"
+#include "storage/value_column.hpp"
+#include "types.hpp"
+#include "utils/enum_constant.hpp"
+
+namespace opossum {
+
+class DeprecatedDictionaryEncoder : public ColumnEncoder<DeprecatedDictionaryEncoder> {
+ public:
+  static constexpr auto _encoding_type = enum_c<EncodingType, EncodingType::DeprecatedDictionary>;
+
+  template <typename T>
+  std::shared_ptr<BaseColumn> _encode(const std::shared_ptr<ValueColumn<T>>& value_column) {
```

Comment:
Assuming that nothing changed here. Please correct me if that's wrong.

Reply:
Yes, nothing changed. I just moved the code.